applications/schedview-snapshot/templates/_helpers.tpl

```diff
@@ -1,3 +1,28 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "schedview-snapshot.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "schedview-snapshot.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
```

Comment:
We're no longer using this pattern with any new applications because it's unnecessary complexity that's never used in Phalanx. I recommend deleting this all again and just putting schedview-snapshot literally in your YAML files.

Reply:
Change made in commit ca02a336.